sklearn/datasets/base.py

```diff
@@ -255,14 +255,25 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
+    if return_X_y == false
```

Comment:
i think you can actually omit this...i'm thinking of the notion that the "default" case is returning the `Bunch`, and you return the tuple `if return_X_y is True`